dev/com.ibm.ws.crypto.passwordutil/resources/com/ibm/ws/crypto/util/internal/resources/Messages.nlsprops

```diff
@@ -50,11 +50,19 @@ PASSWORDUTIL_UNSUPPORTEDENCODING_EXCEPTION.useraction=Check the server log and i
 PASSWORDUTIL_INVALID_BASE64_STRING=CWWKS1859E: The password was not decrypted because a decoding error was reported.
 PASSWORDUTIL_INVALID_BASE64_STRING.explanation=The encrypted password was not encoded properly.
 PASSWORDUTIL_INVALID_BASE64_STRING.useraction=Make sure that the encrypted password is not truncated and encoded by using Base64 encoding.
+PASSWORDUTIL_EXCEPTION_FIPS130_AES128_UNAVAILABLE_ALGORITHM=CWWKS1860E: AES-128 encrypted passwords cannot be used when FIPS 140-3 is enabled. Regenerate the password with AES-256 using the securityUtility command.
+PASSWORDUTIL_EXCEPTION_FIPS130_AES128_UNAVAILABLE_ALGORITHM.explanation=FIPS-130 is enabled and does not support AES-128. Use the securityUtility command to encrypt the password using AES-256.
+PASSWORDUTIL_EXCEPTION_FIPS130_AES128_UNAVAILABLE_ALGORITHM.useraction=Make sure the password is encrypted with AES-256.
+PASSWORDUTIL_EXCEPTION_UNAVAILABLE_ALGORITHM=CWWKS1861E: AES password decryption failed. Check your java.security settings and add the missing algorithm.
+PASSWORDUTIL_EXCEPTION_UNAVAILABLE_ALGORITHM.explanation=An encryption algorithm is being used that is not defined in your java.security.
```

Comment:
To simplify "is being used that", you might change the explanation to either of the following:
- An encryption algorithm in use is not defined in your java.security file.
- An encryption algorithm is missing from your java.security file.

Reply:
Fixed in https://github.com/OpenLiberty/open-liberty/pull/30675/commits/24350ead87d7a491c059e2d2df1d078062609d21